.env.example

```diff
@@ -55,3 +57,9 @@ AML_REBUILD_ENVIRONMENT = 'false'
 # Smoke test image URLs and Classes separated by comma
 TEST_IMAGE_URLS = 'url_to_sunflowers,url_to_daisy'
 TEST_IMAGE_CLASSES = 'sunflowers,daisy'
+
+# Processing OS cmd on custom docker image
+AML_CUSTOM_DOCKER_ENV_NAME = 'custom_docker_env'
+AML_CUSTOM_DOCKERFILE = 'ml_model/preprocessing/Dockerfile'
```

Comment:
May be worth specifying that this AML_CUSTOM_DOCKERFILE is for preprocessing (in case other custom dockerfiles want to be included as well for other steps.

Reply:
Thanks, that's right. I will change it.

In general we are discussing on how far we want to go with the current 'variable' scheme. I personally think that the current structure overdoes is slightly. Things like the relative path to a Dockerfile or a Python script file used in the AML Pipeline creation script does not seem to be something we want to be parameterized. Especially I find it hard to code (add new variable/parameter to the Env Helper, than to .env.example and finally to .env for local testing, then if you did it wrong modify several places in different files) also looking up the value in a PR review is going back an forth.

For sure there are things we want to keep as parameters, e.g. common parameters in all pipeline creation scripts, infrastructure related information (e.g. sizing of the aml compute cluster).

Reply:
@tarockey 
Changed some other variable names as well. Please have a look!